clients/drcachesim/reader/compressed_file_reader.cpp

```diff
@@ -32,6 +32,34 @@
 
 #include "compressed_file_reader.h"
 
+bool
+open_single_file_common(const std::string &path, gzFile &out)
```

Comment:
These are not in any class namespace nor are they static.  Let's at least make them static or possibly take a step toward #4343 and add the dynamorio::drmemtrace namespace.

Reply:
Done.